json_theme/lib/src/codec/theme_encoder.dart

```diff
@@ -6093,7 +6093,7 @@ class ThemeEncoder {
   ///  * [encodeTabAlignment]
   ///  * [encodeTabBarIndicatorSize]
   ///  * [encodeTextStyle]
-  static Map<String, dynamic>? encodeTabBarTheme(TabBarTheme? value) {
+  static Map<String, dynamic>? encodeTabBarTheme(TabBarThemeData? value) {
```

Comment:
This should be named `encodeTabBarThemeData` since it now takes a `ThemeBarData`.

Also, curious why there isn't a matching ThemeDecoder change.